eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/README.md

```diff
@@ -4,227 +4,124 @@ languages:
 - java
 products:
 - azure-event-hubs
-description: "Azure Spring Cloud Stream Binder Sample project for Event Hub client library"
-urlFragment: "azure-spring-cloud-sample-eventhubs-binder"
+description: "Spring Cloud Azure Stream Binder Sample project for Event Hubs client library"  
+urlFragment: "spring-cloud-azure-stream-binder-sample-eventhubs"
 ---
 
-# Azure Spring Cloud Stream Binder for Event Hub Code Sample shared library for Java
+# Spring Cloud Azure Stream Binder for Event Hubs Sample shared library for Java
 
 ## Key concepts
 
-This code sample demonstrates how to use the Spring Cloud Stream Binder for Azure Event Hub.The sample app has two operating modes.
+This code sample demonstrates how to use the `Spring Cloud Stream Binder` for `Azure Event Hubs`.The 
+sample app has two operating modes.
 One way is to expose a Restful API to receive string message, another way is to automatically provide string messages.
-These messages are published to an event hub. The sample will also consume messages from the same event hub.
+These messages are published to one `Event Hub` instance and then consumed by one consumer 
+endpoint from the same application.
 
 ## Getting started
 
 Running this sample will be charged by Azure. You can check the usage and bill at 
-[this link][azure-account].
+[here][azure-account].
 
+### Create Azure resources
 
+1. Create [Azure Event Hubs][create-event-hubs].
+    Please note `Basic` tier is unsupported. After creating the Azure Event Hub, you
```

Comment:
remove this description of "Please note `Basic` tier is unsupported."

Reply:
updated